CHANGELOG.md

```diff
@@ -36,6 +36,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1809])(https://github.com/open-telemetry/opentelemetry-python/pull/1809)
 - Fixed sequence values in OTLP exporter not translating
   ([#1818](https://github.com/open-telemetry/opentelemetry-python/pull/1818))
+- Update get_tracer to return an empty string when passed an invalid name
```

Comment:
This should be added to the "Unreleased" section of the changelog

Reply:
Just did it. Thanks!